Sources/PackageDescription/Target.swift

```diff
@@ -505,9 +520,68 @@ public final class Target {
     ///   - swiftSettings: The Swift settings for this target.
     ///   - linkerSettings: The linker settings for this target.
     ///   - plugins: The plugins used by this target.
-    @available(_PackageDescription, introduced: 5.5)
+    @available(_PackageDescription, introduced: 5.5, obsoleted: 5.9)
```

Comment:
Do we actually need to do this or can we add the new argument with a default value? We do not care about ABI for `PackageDescription` since we're always building clients from source.

Reply:
Actually, we do need this because of availability, right?

Reply:
yes we need this per comment above